File: src/Goods.Domain/LocalAreas/Enums/AreaType.cs
using System.ComponentModel.DataAnnotations;

namespace Goods.Domain.LocalAreas.Enums;

public enum AreaType {
	[Display(Name = "Город")]
    City = 1,
	[Display(Name = "Деревня")]
    Village = 2,
	[Display(Name = "Сельская местность")]
    CountrySide = 3,
}

File: src/Areas.BackOffice/client-app/src/domain/localAreas/enums/areaType.ts
export enum AreaType {
  City = 1,
  Village = 2,
  CountrySide = 3,
}
export namespace AreaType {
  export const getDisplayName = (areaType: AreaType): string => {
    switch (areaType) {
      case AreaType.City:
        return "Город";
      case AreaType.Village:
        return "Деревня";
      case AreaType.CountrySide:
        return "Сельская местность";
    }
  };
}
